drafts/2019-11-06-1900011.adoc

```diff
@@ -134,7 +137,7 @@ For more information:
 
 
 [#hm21]
-== Monitor microservice applications easily wth metrics (MicroProfile Metrics 2.1 )
+== Monitor microservice applications easily wth metrics 
```

Comment:
"... (MicroProfile Metrics 2.2)" (or whatever the version number is in the issue I forwarded to you by email).